contracts/nois-proxy/src/contract.rs

```diff
@@ -104,6 +118,9 @@ pub fn execute(
             amount,
             address,
         } => execute_withdraw(deps, env, info, denom, amount, address),
+        ExecuteMsg::UpdateAllowList { add, remove } => {
+            execute_update_allowlist(deps, env, info, &add, &remove)
```

Comment:
Let's pass those vectors owned. Maybe this is useful later on.

```suggestion
            execute_update_allowlist(deps, env, info, add, remove)
```